pkg/bpf/map_linux.go

```diff
@@ -446,48 +442,27 @@ func (m *Map) setPathIfUnset() error {
 	return nil
 }
 
-// EndParallelMode ends the parallel mode of a map
-func (m *Map) EndParallelMode() {
-	m.lock.Lock()
-	defer m.lock.Unlock()
-
-	if m.inParallelMode {
-		m.inParallelMode = false
-		m.scopedLogger().Debug("End of parallel mode")
-	}
-}
-
-// OpenParallel is similar to OpenOrCreate() but prepares the existing map to
-// be faded out while a new map is taking over. This can be used if a map is
-// shared between multiple consumers and the context of the shared map is
-// changing. Any update to the shared map would impact all consumers and
-// consumers can only be updated one by one. Parallel mode allows for consumers
-// to continue using the old version of the map until the consumer is updated
-// to use the new version.
-func (m *Map) OpenParallel() (bool, error) {
+// Recreate removes any pin at the Map's pin path, recreates and re-pins it.
+func (m *Map) Recreate() error {
```

Comment:
Hmm, why not do a real charry-pick of https://github.com/ti-mo/cilium/commit/534160afb97d761e987158be5ba71116e5ed5818 or other needed commits from the branch? That way my rebase will be less painful, and I also get some credit. :)